tests/FileManipulation/UnusedVariableManipulationTest.php

```diff
@@ -348,14 +348,27 @@ function foo() : void {
                 '<?php
                     function foo() : void {
                         $a = 5;
-                        $b = &$a;
```

Comment:
weird, this test was basically testing nothing...

Reply:
Yep, it was confirming incorrect behavior...
Unfortunately the actual correct behavior doesn't work yet either, but I figure I'll leave it for a future improvement.

Reply:
I hate that kind of tests when they're not flagged properly. When you break them, you just have no idea if the test is legit or not

Reply:
Agreed, but at least this one was obvious.